cypress/integration/raise-repair/form.spec.js

```diff
@@ -170,7 +165,12 @@ describe('Raise repair form', () => {
         'href',
         '/properties/00012345'
       )
-      cy.contains('Start a new search')
+      cy.contains('Start a new search').should('have.attr', 'href', '/')
+      cy.contains('View work order').should(
+        'have.attr',
+        'href',
+        '/work-orders/10102030'
+      )
```

Comment:
how about wrapping that in an `it`?

Reply:
Hey not sure what you mean by this. Another spec are you implying?